packages/cli/src/commands/init/init.js

```diff
@@ -104,6 +107,87 @@ async function createFromTemplate({
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
+
+    const {shouldInstallCocoaPods} = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'shouldInstallCocoaPods',
+        message: 'CocoaPods is not installed, do you want to install it?',
+      },
+    ]);
+
+    if (shouldInstallCocoaPods) {
+      try {
+        // First attempt to install `cocoapods`
+        await execa('gem', ['install', 'cocoapods'], {
+          stdio: 'pipe',
+        });
+      } catch (err) {
+        try {
+          // If that doesn't work then try with sudo
+          await execa('sudo', ['gem', 'install', 'cocoapods'], {
+            stdio: 'pipe',
+          });
+        } catch (err) {
+          throw new CLIError(
+            'Error occurred while trying to install CocoaPods, please run this command again.',
+            err,
+          );
+        }
+      }
+
+      loader.start('Installing pods');
+    }
+  }
+
+  try {
+    await execa('pod', ['install'], {
+      stdio: 'pipe',
+    });
+  } catch (err) {
+    throw new CLIError(
+      'Failed to run "pod install", please try to run it manually.',
+      err,
+    );
+  }
+}
+
+async function installDependencies({
+  npm,
+  loader,
+}: {
+  npm?: boolean,
+  loader: typeof Ora,
+}) {
+  loader.start('Installing all required dependencies');
+
+  await PackageManager.installAll({
+    preferYarn: !npm,
+    silent: true,
+  });
+
+  if (process.platform === 'darwin') {
+    await installPods(loader);
+
+    process.chdir('..');
```

Comment:
Sorry, I don't follow, what do you mean?

Reply:
At current state, if somebody changes the implementation of `installPods` to not change dir to `./ios`, and will simply forget to change this line (because it's outside of the fn) we'll end up changing directory to something unintended. 

What I want is to collocate `process.chdir` calls, so they happen in the same code/function block. Either:

```js
// process.chdir calls are close to each other:
try {
  process.chdir('./ios');
  await installPods(loader)
} catch(error) {
  throw error
} finally {
  process.chdir('..')
}
```

or 
```js
function installPods(loader) {
 try {
  process.chdir('./ios');
  // do stuff...
 } catch(error) {
  throw error
 } finally {
  process.chdir('..')
 }
}
```

Reply:
Ah I see, that's awesome, I'll do it.

Thanks!